glotaran/parse/test/test_model_parser_kinetic.py

```diff
@@ -93,8 +93,8 @@ def test_initial_concentration(model):
     assert len(model.initial_concentration) == 2
 
     i = 1
+    label = f"inputD{i}"
     for _ in model.initial_concentration:
-        label = f"inputD{i}"
```

Comment:
If it was ment to be 
```python
for i, _ in enumerate(model.initial_concentration, start=1):
    label = f"inputD{i}"
    assert label in model.initial_concentration
```
This might be a buggy test

Reply:
That's a minor bug in the test case. There are two inputs that are being tested, inputD1 and inputD2, so the for loop line should be replaced with:
`for i, _ in enumerate(model.initial_concentration, start=1):`

Reply:
Committed exactly that fix (forgot to refresh page so didn't see the updated comment ^^)